example/pom.xml

```diff
@@ -132,6 +143,27 @@ Also contains a sample Docker Compose file (see &quot;docker&quot; folder) for s
             <groupId>io.fabric8</groupId>
             <artifactId>fabric8-maven-plugin</artifactId>
           </plugin>
+          <plugin>
+            <artifactId>maven-assembly-plugin</artifactId>
+            <version>3.0.0</version>
+            <executions>
+              <execution>
+                <id>eclipse-hono</id>
+                <phase>package</phase>
+                <goals>
+                    <goal>single</goal>
+                </goals>
+                <configuration>
+                  <descriptors>
+                    <descriptor>src/main/assembly/hono.xml</descriptor>
+                  </descriptors>
+                  <tarLongFileMode>posix</tarLongFileMode>
+                  <finalName>eclipse-hono-${project.version}</finalName>
```

Comment:
if we stay that it's still an "example", maybe it should be reflected in the final tar name ? i.e. eclipse-hono-example-${project.version} ?